app/javascript/components/public_health/PatientsTable.js

```diff
@@ -578,6 +579,10 @@ class PatientsTable extends React.Component {
                         <i className="fas fa-clipboard-list text-center" style={{ width: '1em' }}></i>
                         <span className="ml-2">Update Case Status</span>
                       </Dropdown.Item>
+                      <Dropdown.Item className="px-3" onClick={() => this.setState({ action: 'Update Assigned User' })}>
+                        <i className="fas fa-user-edit text-center" style={{ width: '1em' }}></i>
```

Comment:
Would it make sense to keep this icon consistent with the icon next to "Change User" on the patient page? I actually like this icon more, would be nice to update the other one along with it as well, but probably something to ask the PO team.

Reply:
I will follow up with the PO team to ask about this and update it based on their feedback.

Reply:
I also think this icon is a better choice - if PO team agrees we should change the other one (here: https://github.com/SaraAlert/SaraAlert/blob/master/app/javascript/components/subject/monitoring_actions/AssignedUser.js#L170)!

Reply:
Just got feedback from PO team, they prefer the original icon so I will update it accordingly.